x-pack/filebeat/module/okta/system/test/okta-system-test.json.log-expected.json

```diff
@@ -74,7 +74,6 @@
         ],
         "user.full_name": "xxxxxx",
         "user_agent.device.name": "Mac",
-        "user_agent.device.type": "Desktop",
```

Comment:
I think you might need to update your docker images before running the tests, or just discard the changes since it seems there are no changes in the output test files.

Reply:
done